src/index.js

```diff
@@ -1,3 +1,11 @@
+import {Mark} from "./mark.js";
+import {plot} from "./plot.js";
+
+// Note: this side effect avoids a circular dependency.
+Mark.prototype.plot = function plotThis({marks = [], ...options} = {}) {
```

Comment:
```suggestion
Mark.prototype.plot = function ({marks = [], ...options} = {}) {
```

I don't think we need to name this function?